extra-radio/aprx/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo 'Append -g to CFLAGS to make this have debug symbols and -fcommon -O2 to compile with gcc 10'
+export CFLAGS="-Wall -g -fcommon -O2"
```

Comment:
You should be using Autobuild3-defined flags, and with that, you'd only need to append `-fcommon`. Do the following:

```
export CFLAGS="${CFLAGS} -fcommon"
```

And adjust your `abinfo` accordingly.

Also, almost all of your files lack `newline` markers - Visual Studio Code?

Reply:
Sorry to make you laugh. I used $CFLAGS before but I didn't keep it for no reason ...
In addition, I used KWrite to make all modification. It's a little wired, because I notice almost every local file has `newline`.